browser.c

```diff
@@ -1606,21 +1606,60 @@ void mutt_buffer_select_file(struct Buffer *file, SelectFileFlags flags,
           mutt_message("%s", ARRAY_GET(&state.entry, menu->current)->name);
         break;
 
-#ifdef USE_IMAP
-      case OP_BROWSER_TOGGLE_LSUB:
-        bool_str_toggle(NeoMutt->sub, "imap_list_subscribed", NULL);
+      case OP_CREATE_MAILBOX:
+      {
+        int create_rc;
+        bool created = false;
+        struct Mailbox *m = mailbox_find(CurrentFolder);
+        /* PATH_MAX used for local mailboxes, e.g. maildir, but should also be
+         * sufficient for remote mailboxes */
```

Comment:
I'm still no sufficiently familiar with the other use cases to be confident in this assumption, so I'd be happy to hear what others think.

Reply:
yes, `PATH_MAX` (usually 4096) should be fine